### DISCUSSION TITLE
coder如何做到租户管理的呢

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/coder#8296] [General]
coder如何做到租户管理的呢？当前具备这个功能吗，具体代码实现是哪个类

**Comment by matifali** [comment]
Please use English.

**Reply by Pythoneryeah** [reply]
How does coder manage tenants? Currently have this function, the specific code implementation is which class

**Comment by matifali** [comment]
Please elaborate. It's not clear what you are trying to ask.
